src/styles/blocks/card.scss

```
@@ -0,0 +1,90 @@
+$bgcolor-bt: #00acdc;
+$color-grey:#616070;
+$color-black: #060b35;
+$fs12: 12px;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin: 33px 19px 0;
+  }
+
+  &__photo {
+    display: block;
+    max-width: 100%;
+    height: auto;
+  }
+
+  &__container {
+    margin: 40px 16px 16px;
+  }
+
+  &__title {
+    padding-bottom: 4px;
+  }
+  &__title-h2 {
+    font-weight: 500;
+    font-size: $fs12;
+    line-height: 18px;
+    margin: 0 ;
+  }
+  &__subtitle {
+    font-size: 10px;
+    line-height: 1.4;
+    color: $color-grey;
+    padding-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: $color-black;
+    padding-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 16px;
+  }
+
+  &__price-left {
+    font-size: $fs12;
+    line-height: 18px;
+    color: $color-grey;
+  }
+
+  &__price-right {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
```

use nesting in all cases
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    align-items: center;
    padding-bottom: 16px;
    
    &-left {
      font-size: $fs12;
      line-height: 18px;
      color: $color-grey;
    }
  
    &-right {
      font-weight: 700;
      font-size: 16px;
      line-height: 18px;
    }
```